ext_localconf.php

```diff
@@ -21,50 +21,43 @@
 $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['FileDumpEID.php']['checkFileAccess']['FalSecuredownload'] =
     \BeechIt\FalSecuredownload\Hooks\FileDumpHook::class;
```

Comment:
This hook doesn't exists anymore. It was deprecated in TYPO3 11 and removed in 12. It should be replaced with the `\TYPO3\CMS\Core\Resource\Event\ModifyFileDumpEvent` PSR-14 event. Without it there's no access check on links to (restricted) files.

Reply:
Yes, you are absolutly right. I missed that. I will migrate the hook to a PSR-14 event. I will check the other hooks as well, too.

Reply:
I missed this pull request and did a pull request myself adding compatibility to v12, also for the ModifyFileDumpEvent: https://github.com/beechit/fal_securedownload/pull/210 . Maybe this could be considered instead?